docs/conf.py

```diff
@@ -1,47 +1,29 @@
 # -*- coding: utf-8 -*-
 #
-# hy documentation build configuration file, created by
-# sphinx-quickstart on Tue Mar 12 22:52:58 2013.
-#
 # This file is execfile()d with the current directory set to its containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
 
-import os
-import sys
-import time
+import os, sys, time, subprocess, runpy, cgi
 sys.path.append(os.path.abspath(".."))
 
-import hy
-
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
-
-# -- General configuration -----------------------------------------------------
-
-# If your documentation needs a minimal Sphinx version, state it here.
-#needs_sphinx = '1.0'
-
-# Add any Sphinx extension module names here, as strings. They can be extensions
-# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.todo']
+# Try to get a version number.
+# N.B. Similar code appears in setup.py.
+# We don't use the dirty flag here because Read the Docs may dirty
+# the repository while generating the documentation.
+VERSIONFILE='../hy/version.py'
```

Comment:
Before this PR, did running `sphinx-build` manually from a different directory work? What's the exact command?